wolfcrypt/src/misc.c

```diff
@@ -796,6 +806,13 @@ WC_MISC_STATIC WC_INLINE w64wrapper w64ShiftLeft(w64wrapper a, int shift)
     return a;
 }
 
+WC_MISC_STATIC WC_INLINE w64wrapper w64Mul(word32 a, word32 b)
+{
+    w64wrapper ret;
+    ret.n = (word64)a * (word64)b;
```

Comment:
You only need to cast one of them to word64.
```
ret.n = (word64)a * b;
```

Reply:
would rather be explicit than rely on the compiler, especially the old compiler that was being used with the pic18f work